### PR TITLE
add support new actions iopay desktop

### LINKED ISSUE #1461: support new actions in iopay desktop
image

this is a vote action. It does not show up in iopay desktop

### PR DESCRIPTION
**Pull request by hunglmtb**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1461 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by hunglmtb** [comment]
image


image



image



image


image



image



image

File: translations/en.yaml [hunk 1]
```diff
@@ -177,7 +177,16 @@ confirmation.method: Method
 confirmation.owner: Owner
 confirmation.price: 'Gas price'
 confirmation.toAddress: 'To addresss'
-confirmation.toContract: 'To contract'
+confirmation.candidateName: 'Candidate Name'
```
Comment (guo) [inline translations/en.yaml:180]: let's just call it Delegate Name

File: translations/en.yaml [hunk 2]
```diff
@@ -177,7 +177,16 @@ confirmation.method: Method
...
+confirmation.stakedAmount: 'Staked Amount'
+confirmation.stakedDuration: 'Staked Duration'
+confirmation.autoStake: 'Auto-Stake'
+confirmation.payload: 'Payload'
+confirmation.bucketIndex: 'Bucket Index'
+confirmation.voterAddress: 'Voter Address'
+confirmation.ownerAddress: 'Owner Address'
+confirmation.candidate: 'Candidate'
```
Comment (guo) [inline translations/en.yaml:188]: use Delegate.

File: src/shared/wallet/decode-contract-data.ts [hunk 3]
```diff
@@ -74,3 +76,26 @@ export function decode(
   window.console.warn("can not found method");
   return { method: data, data: undefined };
 }
+
+export function decodeData(
+  data: string | Buffer,
+  contractAddress: string = "io1p99pprm79rftj4r6kenfjcp8jkp6zc6mytuah5"
```
Comment (guo) [inline src/shared/wallet/decode-contract-data.ts:82]: what is this?

Comment (hunglmtb) [reply src/shared/wallet/decode-contract-data.ts:82]: it's the default value of contract address when caller no input this param, similar to https://github.com/iotexproject/iotex-explorer/blob/fe8fa989d430c02bb280e104bb966a4f4cc8a7b9/src/shared/wallet/decode-contract-data.ts#L17-L21

File: translations/en.yaml [hunk 4]
```diff
@@ -177,7 +177,16 @@ confirmation.method: Method
...
-confirmation.toContract: 'To contract'
+confirmation.candidateName: 'Delegate Name'
+confirmation.stakedAmount: 'Staked Amount'
+confirmation.stakedDuration: 'Staked Duration'
+confirmation.autoStake: 'Auto-Stake'
+confirmation.payload: 'Payload'
+confirmation.bucketIndex: 'Bucket Index'
+confirmation.voterAddress: 'Voter Address'
+confirmation.ownerAddress: 'Owner Address'
+confirmation.candidate: 'Delegate'
```
Comment (guo) [inline translations/en.yaml:188]: where is it used?

**Pull request merged**